.github/workflows/publish-release.yml

```diff
@@ -2,6 +2,7 @@ name: Publish package to Release Artifactory
 on:
   release:
     types: [created]
+  workflow_dispatch: {}
```

Comment:
This is already manually triggered by creating a github release. In case of GHA hanging and we want to try, what would be the preferred process? Manually create a github release or direct launch this GHA. Feels former is better and more trackable?